packages/contractkit/src/wrappers/Election.ts

```diff
@@ -110,23 +113,43 @@ export class ElectionWrapper extends BaseWrapper<Election> {
     this.contract.methods.getGroupsVotedForByAccount
   )
 
-  async getVotesForGroupByAccount(account: Address, group: Address): Promise<GroupVote> {
-    const pending = await this.contract.methods
-      .getPendingVotesForGroupByAccount(group, account)
-      .call()
-    const active = await this.contract.methods
-      .getActiveVotesForGroupByAccount(group, account)
-      .call()
+  async getVotesForGroupByAccount(
+    account: Address,
+    group: Address,
+    blockNumber?: number
+  ): Promise<GroupVote> {
+    const pending = blockNumber
```

Comment:
same as before... you rpbably don't need the conditonal.. 
for this and all the others